.devcontainer/Containerfile

```diff
@@ -0,0 +1,8 @@
+FROM registry.access.redhat.com/ubi8/go-toolset:1.22
```

Comment:
should we use ubi8 or ubi9 ? Also does it have by default make tools?

Reply:
Yes, it does have

Reply:
@praveenkumar I am OK with either option. I have been building on my machine with this setup for a long time now; never needed to target EL8, actually. As long as it does not cause issues with `glibc` I can switch to UBI9.